backend/runner/runner.go

```diff
@@ -52,6 +51,7 @@ type Config struct {
 	Language              []string        `short:"l" help:"Languages the runner supports." env:"FTL_LANGUAGE" default:"go,kotlin,rust"`
 	HeartbeatPeriod       time.Duration   `help:"Minimum period between heartbeats." default:"3s"`
 	HeartbeatJitter       time.Duration   `help:"Jitter to add to heartbeat period." default:"2s"`
+	RunnerStartDelay      int             `help:"Time in seconds for a runner to wait before contacting the controller. This can be needed in istio environments to work around initialization races." env:"FTL_RUNNER_START_DELAY" default:"0"`
```

Comment:
This should be a `time.Duration` - it will automatically parse values like `1s`, `1m`, etc.

Reply:
(we'll need to update any locations where FTL_RUNNER_START_DELAY is set to an unadorned integer value though, but it's better for this to be correct now than have to fix it later)

Reply:
@alec-brooks I have updated to PR to fix this, but we will need to update the infra repo once this change is deployed.